narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,220 @@
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [The mutation rate of SARS-CoV-2 is typical for a coronavirus](https://nextstrain.org/ncov/global/2020-05-14?c=num_date&d=tree,entropy&l=clock&p=full)  
+
+Because mutations are biologically inevitable, they tend to accumulate at a steady rate over time.
+<br><br>
+Here, the x axis shows the date that each sample was collected. The y axis shows the *total number* of mutations away from the root of the tree. Each sample is colored by which region of the world it was collected in.
+<br><br>
+While there are certainly some outliers, on average, we see changes accumulate at a rate of ~24 substitutions per year. This means that if a single viral lineage were transmitting from one person to the next for a full year, we'd expect it to accumulate roughly 24 substitutions by the end of the year. Importantly, this corresponds to only (24/30,000)\*100\% = ~0.08\% of the viral genome changing each year.
+<br><br>
+For context, SARS (also a coronavirus) accumulates XX substitutions per year (changing XX\% of its genome), while influenza accumulates XX substitutions per year (changing XX\% of its genome).
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [As far as we know, there is only 1 strain of SARS-CoV-2](https://nextstrain.org/ncov/global/2020-05-14?branchLabel=clade&c=clade_membership&d=tree&p=full)  
+
+There have been many reports about multiple "strains" of SARS-CoV-2.
+First, a clarification: when virologists use the word "strain," it often just refers to a useful label so we can all refer to the same group of samples (e.g., our clade labels shown here). This does *not* imply any known functional difference.
+<br><br>
+Separately, "strain" can be used to refer to viral genotypes that are functionally distinct, either biologically (e.g., pathogenicity/disease severity) and/or epidemiologically (e.g., transmissibility).
+Importantly, though, determining whether two genotypes actually *are* functionally distinct requires much more experimental, clinical and epidemiological data than we currently have.
+<br><br>
+One of the most prominent hypotheses about SARS-CoV-2 strains compares possible "D614" vs "G614" strains. We can use Nextstrain to investigate.
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text -->
+# [D614G may be related to transmissibility, but could also be explained by geography](https://nextstrain.org/ncov/global/2020-05-14?c=gt-S_614&gmax=25778&gmin=21082&p=full)  
+
+[A recent paper](https://www.biorxiv.org/content/10.1101/2020.04.29.069054v1) suggests that a single mutation, D614G (from amino acid "D" to "G" at site 614), may be responsible for increasing the transmissibility and/or pathogenicity of SARS-CoV-2.
```

Comment:
```suggestion
[A recent pre-print](https://www.biorxiv.org/content/10.1101/2020.04.29.069054v1) suggests that a single mutation, D614G (from amino acid "D" to "G" at site 614 in the "S" ('spike') protein), may be responsible for increasing the transmissibility and/or pathogenicity of SARS-CoV-2.
```
I think mostly they talk about transmissibility though - not pathogenicity. Haven't read paper recently enough to be 100% of this, but worth checking & maybe removing if it's mostly transmit.

Reply:
Yes, primarily about transmissibility, they don't find any evidence of clinical outcome changes.